docs/guidelines/language/basics/grammar.md

```
@@ -0,0 +1,265 @@
+- When you delete this file, it cannot be recovered.
+
+</div>
+</div>
+
+## Related
+
+- [Toast messages](../messaging/toast-messages.mdx)
+- [Error messages](../messaging/error-messages.mdx)
+- [Warning messages](../messaging/warning-messages.mdx)
+- [License management](../menu-functions-and-ui-labels/license-management.md)
+- [Localization](#)
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)